workflows/cloudify_system_workflows/deployment_environment.py

```diff
@@ -158,6 +168,11 @@ def create(ctx, labels=None, inputs=None, skip_plugins_validation=False,
     idd.create(ctx, client, deployment_plan)
 
 
+def _get_deployments(client: 'CloudifyClusterClient',
```

Comment:
doesn't need to be cluster, just `CloudifyClient` is fine (which is a superclass of cluster)